File: app/src/main/java/com/app/test/data/network/HttpResponse.java
package com.app.test.data.network;

/**
 * Created by Sweven on 2023/8/29--14:20.
 * Email: sweventears@163.com
 */
public class HttpResponse<T> {
    private String message;
    private T data;
    private int code;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }
}

File: app/src/main/java/com/app/test/data/network/HttpResponse.kt
package com.app.test.data.network

/**
 * Created by Sweven on 2025/9/12--20:39.
 * Email: sweventears@163.com
 */
open class HttpResponse<T> @JvmOverloads constructor(
    var code: Int = 0,
    var message: String? = null,
    var data: T? = null,
)